jina/peapods/pea.py

```diff
@@ -367,7 +370,7 @@ def post_init(self):
         """
         pass
 
-    def close(self) -> None:
+    def send_terminate_signal(self) -> None:
```

Comment:
please keep `close()`, as it is the convention `of context_manager`. It also allows people to manually close a CM without using `with`.

Suggest:

```python
def __exit__(self, exc_type, exc_val, exc_tb) -> None:
       self.close()

def close(self) -> None:
        self.send_terminate_signal()
        self.is_shutdown.wait()
             clear_queue()
             self.join()
```

Reply:
Done!